solr/solrj/src/java/org/apache/solr/common/params/CommonParams.java

```diff
@@ -289,6 +289,14 @@ public static EchoParamStyle get(String v) {
   @Deprecated(since = "9.4")
   String DISABLE_REQUEST_ID = "disableRequestId";
 
+  /**
+   * Parameter to disable the distributed term statistics request for current query when distributed
+   * IDF is enabled in solrconfig
+   *
+   * <p>Defaults to 'false' if not specified
+   */
+  String DISABLE_DISTRIB_STATS = "disableDistribStats";
```

Comment:
~~It's time for bikeshedding~~ Giving the existing parameter `distrib.singlePass`, I think we should name it as  `distrib.statsCache=false`. Where `statsCache` repeats the tag in solrconfig.xml.

Reply:
changed to distrib.statsCache